### PR TITLE
apt*, dotnet*: add Spanish translation

### PR DESCRIPTION
**Pull request by Nacho-source**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/dotnet-publish.md [hunk 1]
```diff
@@ -26,3 +26,4 @@
 - Specify the output directory:
 
 `dotnet publish --output {{path/to/directory}} {{path/to/project_file}}`
+
```
Comment (schneiderl) [inline pages/common/dotnet-publish.md:29]: I believe this change was made by mistake, right?

File: pages/common/dotnet-restore.md [hunk 2]
```diff
@@ -26,3 +26,4 @@
 - Restore dependencies with a specific verbosity level:
 
 `dotnet restore --verbosity {{quiet|minimal|normal|detailed|diagnostic}}`
+
```
Comment (schneiderl) [inline pages/common/dotnet-restore.md:29]: I believe this change was made by mistake, right?

File: pages/common/dotnet.md [hunk 3]
```diff
@@ -18,3 +18,4 @@
 - Run a packaged dotnet application (only needs the runtime, the rest of the commands require the .NET Core SDK installed):
 
 `dotnet {{path/to/application.dll}}`
+
```
Comment (schneiderl) [inline pages/common/dotnet.md:21]: I believe this change was made by mistake, right?

**Review by schneiderl — CHANGES_REQUESTED** [review]
Hi, thank you for the PR 👍 
I don't know the language so I'm not reviewing the Spanish files (only reviewing it syntax wise).
But there are changes on the English files which I'm assuming were made by mistake.
Could you please take a look at that? 

Thanks a lot for your translations, it really helps a lot.

**Review by schneiderl — APPROVED** [review]
Looks good to me now. Thank you for your PR. 👍

File: pages.es/common/dotnet-publish.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+> Publica una aplicación .NET y sus dependencias en una carpeta para despliegue de un sistema local.
+> Más información: https://docs.microsoft.com/dotnet/core/tools/dotnet-publish.
+
+- Compila un proyecto .NET en modo de lanzamiento:
+
+`dotnet publish --configuration Release {{ruta/al/archivo_del_proyecto}}`
+
+- Publica el tiempo de ejecución de .NET Core con la aplicación para el tiempo de ejecución especificado:
+
+`dotnet publish --self-contained true --runtime {{identificador_del_tiempo_de_ejecución}} {{ruta/al/archivo_del_proyecto}}`
```
Comment (navarroaxel) [inline pages.es/common/dotnet-publish.md:12]:
```suggestion
`dotnet publish --self-contained true --runtime {{identificador_del_entorno_en_tiempo_de_ejecución}} {{ruta/al/archivo_del_proyecto}}`
```

File: pages.es/common/dotnet-publish.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Empaqueta la aplicación en un archivo ejecutable unico de una plataforma específica:
+
+`dotnet publish --runtime {{identificador_del_tiempo_de_ejecucución}} -p:PublishSingleFile=true {{ruta/al/archivo_del_proyecto}}`
```
Comment (navarroaxel) [inline pages.es/common/dotnet-publish.md:16]:
```suggestion
`dotnet publish --runtime {{identificador_del_entorno_en_tiempo_de_ejecucución}} -p:PublishSingleFile=true {{ruta/al/archivo_del_proyecto}}`
```

File: pages.es/common/dotnet-publish.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# dotnet publish
+
+> Publica una aplicación .NET y sus dependencias en una carpeta para despliegue de un sistema local.
+> Más información: https://docs.microsoft.com/dotnet/core/tools/dotnet-publish.
+
+- Compila un proyecto .NET en modo de lanzamiento:
+
+`dotnet publish --configuration Release {{ruta/al/archivo_del_proyecto}}`
```
Comment (navarroaxel) [inline pages.es/common/dotnet-publish.md:8]: The argument options should be between `{{}}`:
```suggestion
`dotnet publish --configuration {{Release}} {{ruta/al/archivo_del_proyecto}}`
```
The same should be applied to the English page.

File: pages.es/common/dotnet-publish.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Publica el tiempo de ejecución de .NET Core con la aplicación para el tiempo de ejecución especificado:
+
+`dotnet publish --self-contained true --runtime {{identificador_del_tiempo_de_ejecución}} {{ruta/al/archivo_del_proyecto}}`
+
+- Empaqueta la aplicación en un archivo ejecutable unico de una plataforma específica:
+
+`dotnet publish --runtime {{identificador_del_tiempo_de_ejecucución}} -p:PublishSingleFile=true {{ruta/al/archivo_del_proyecto}}`
+
+- Recorta librerías no usadas para reducir el tamaño de la aplicación desplegada:
```
Comment (navarroaxel) [inline pages.es/common/dotnet-publish.md:18]: The Spanish translation for `libraries` is `bibliotecas`:
```suggestion
- Recorta las bibliotecas no usadas para reducir el tamaño de la aplicación:
```

File: pages.es/common/dotnet-publish.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`dotnet publish --self-contained true --runtime {{identificador_del_tiempo_de_ejecución}} -p:PublishTrimmed=true {{ruta/al/archivo_del_proyecto}}`
```
Comment (navarroaxel) [inline pages.es/common/dotnet-publish.md:20]:
```suggestion
`dotnet publish --self-contained true --runtime {{identificador_del_entorno_de_tiempo_de_ejecución}} -p:PublishTrimmed={{true}} {{ruta/al/archivo_del_proyecto}}`
```

File: pages.es/common/dotnet-restore.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# dotnet restore
+
+> Restarua las dependencias y herramientas de un proyecto .NET.
+> Más información: https://docs.microsoft.com/dotnet/core/tools/dotnet-restore.
```
Comment (navarroaxel) [inline pages.es/common/dotnet-restore.md:4]:
```suggestion
> Más información: <https://docs.microsoft.com/dotnet/core/tools/dotnet-restore>.
```

File: pages.es/common/dotnet-restore.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+- Restaura dependencias para un proyecto o solución .NET en el directorio actual:
+
+`dotnet restore`
+
+- Restaura dependencias para un proyecto o solución .NET en una ubicación específica:
+
+`dotnet restore {{ruta/al/proyecto_o_solución}}`
+
+- Restaura depedencias sin tomar las solicitudes HTTP:
```
Comment (navarroaxel) [inline pages.es/common/dotnet-restore.md:14]:
```suggestion
- Restaura depedencias sin almacenar las solicitudes HTTP en caché:
```

File: pages.es/common/dotnet-restore.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+`dotnet restore --no-cache`
+
+- Obliga a todas las dependencias a ser resueltas incluso si la última restauración fue exitosa:
+
+`dotnet restore --force`
+
+- Restaura dependencias usando las fallas del paquete fuente como advertencias:
```
Comment (navarroaxel) [inline pages.es/common/dotnet-restore.md:22]:
```suggestion
- Restaura dependencias usando los orígenes con error como advertencias:
```

File: pages.es/common/dotnet-restore.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+`dotnet restore --ignore-failed-sources`
+
+- Restaura dependencias con un nivel específico de verbosidad:
+
+`dotnet restore --verbosity {{el|mínimo|normal|diagnóstico|detallado}}
```
Comment (navarroaxel) [inline pages.es/common/dotnet-restore.md:28]:
```suggestion
`dotnet restore --verbosity {{quiet|minimal|normal|detailed|diagnostic}}
```

File: pages.es/common/dotnet.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
+# dotnet
+
+> Herramienta multiplataforma de línea de comandos para .NET Core.
+> Más información en: https://docs.microsofot.com/dotnet/core/tools
```
Comment (navarroaxel) [inline pages.es/common/dotnet.md:4]:
```suggestion
> Más información: <https://docs.microsofot.com/dotnet/core/tools>.
```

File: pages.es/common/dotnet.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+
+- Inicializa un proyecto .NET nuevo:
+
+`dotnet new {{nombre_corto_modelo}}`
```
Comment (navarroaxel) [inline pages.es/common/dotnet.md:8]:
```suggestion
`dotnet new {{nombre_de_la_plantilla}}`
```

File: pages.es/common/dotnet.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+
+- Restaura los paquetes NuGet:
+
+`dotnet restore`
+
+- Construye y ejectura el proyecto .NET en el directorio actual:
```
Comment (navarroaxel) [inline pages.es/common/dotnet.md:14]:
```suggestion
- Compila y ejectura el proyecto .NET en el directorio actual:
```

File: pages.es/common/dotnet.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+
+`dotnet run`
+
+- Ejecuta una aplicación dotnet empaquetada (solo necesita el tiempo de ejecución, el resto de los comandos requieren el SDK de .NET Core instalado):
```
Comment (navarroaxel) [inline pages.es/common/dotnet.md:18]:
```suggestion
- Ejecuta una aplicación dotnet empaquetada (solo necesita el entorno en tiempo de ejecución, el resto de los comandos requieren el SDK de .NET Core instalado):
```

File: pages.es/common/dotnet.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+
+`dotnet {{ruta/a/aplicación.dll}}`
```
Comment (navarroaxel) [inline pages.es/common/dotnet.md:20]:
```suggestion
`dotnet {{ruta/a/la/aplicación.dll}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.es/linux/apt-cache.md [hunk 18]
```diff
@@ -1,6 +1,6 @@
 # apt-cache
 
-> Herramienta de consulta de paquete de Debian y Ubuntu.
+> Herramienta de consulta de paquetes de Debian y Ubuntu.
```
Comment (navarroaxel) [inline pages.es/linux/apt-cache.md:3]:
```suggestion
> Herramienta de consulta de paquetes para Debian y Ubuntu.
```

File: pages.es/linux/apt-file.md [hunk 19]
```diff
@@ -1,15 +1,15 @@
 # apt-file
 
-> Search for files in apt packages, including ones not yet installed.
+> Busca archivos en paquetes apt, inluidos los que aún no fueron instalados.
```
Comment (navarroaxel) [inline pages.es/linux/apt-file.md:3]:
```suggestion
> Busca archivos en paquetes apt, incluyendo los que aún no fueron instalados.
```

File: pages.es/linux/apt.md [hunk 20]
```diff
@@ -11,26 +11,26 @@
 
 `apt search {{paquete}}`
 
-- Mostrar información para un paquete:
+- Muestra información para un paquete:
```
Comment (navarroaxel) [inline pages.es/linux/apt.md:14]:
```suggestion
- Muestra la información de un paquete:
```

File: pages.es/linux/cmus.md [hunk 21]
```diff
@@ -3,19 +3,19 @@
...
 
 `cmus {{ruta/al/directorio}}`
 
-- Añadir un archivo/directorio a la librería
+- Añade un archivo/directorio a la librería
 
 `:add {{ruta/al/archivo_o_directorio}}`
 
-- Pausar/reproducir la canción actual:
+- Pausa/reproduci la canción actual:
```
Comment (navarroaxel) [inline pages.es/linux/cmus.md:14]:
```suggestion
- Pausa/reproduce la canción actual:
```

File: pages.es/linux/line.md [hunk 22]
```diff
@@ -2,6 +2,6 @@
 
 > Lee una única línea de entrada.
 
-- Lee entrada:
+- Lee una  entrada:
```
Comment (navarroaxel) [inline pages.es/linux/line.md:5]: Doble space
```suggestion
- Lee una entrada:
```

File: pages.es/linux/apt.md [hunk 23]
```diff
@@ -11,26 +11,26 @@
...
 
 `apt show {{paquete}}`
 
-- Instalar un paquete o actualizarlo a su última versión disponible:
+- Instala un paquete o lo actualiza a su última versión disponible:
 
 `sudo apt install {{paquete}}`
 
-- Eliminar un paquete (si se utiliza `purge` también se elimina sus archivos de configuración):
+- Elimina un paquete (si se utiliza `purge` también se eliminan sus archivos de configuración):
```
Comment (navarroaxel) [inline pages.es/linux/apt.md:22]:
```suggestion
- Elimina un paquete (si se utiliza `purge` también elimina sus archivos de configuración):
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.es/linux/lsb_release.md [hunk 24]
```diff
@@ -1,6 +1,6 @@
 # lsb_release
 
-> Proporciona información sobre LSB (Linux Standard Base) y específica de la distribución.
+> Proporciona cierta LSB (Linux Standard Base) e información específica de la distribución.
```
Comment (navarroaxel) [inline pages.es/linux/lsb_release.md:3]:
```suggestion
> Proporciona información específica de la distribución y LSB (Linux Standard Base).
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**